src/placement/jump_map.c

```diff
@@ -1059,6 +1059,7 @@ jump_map_query(struct pl_map *map, struct pl_map_attr *attr)
  *
  * \param[in]   map             A reference to the placement map being used to
  *                              place the object shard.
+ * \param[in]   layout_gl_version layout global version.
```

Comment:
why  emphasize it is "global" version? is it possibly with different "local" version for a obj on different nodes? or just rename it to "layout_ver".

and just confirm the layout change we made recently to fix the shard co-location should with different layout_gl_version?

Reply:
yes, it should not use global, a bit confusion here indeed. 

Yes, any layout change should bump the layout version.